packages/mui-lab/src/internal/pickers/hooks/usePickerState.ts

```diff
@@ -43,6 +43,13 @@ export function usePickerState<TInput, TDateValue>(
 
   const utils = useUtils();
   const { isOpen, setIsOpen } = useOpenState(props);
+  const [initialDate, setInitialDate] = React.useState(props.value);
```

Comment:
```suggestion
  const [initialDate, setInitialDate] = React.useState(value);
```

`props` is already de-structured in line 42.